delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/serialization/EventSerializer.scala

```diff
@@ -0,0 +1,76 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.serialization
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.SerializerWithStringManifest
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.BlazegraphViews
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.model.{BlazegraphViewEvent, BlazegraphViewType, BlazegraphViewValue, ViewRef}
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.serialization.EventSerializer.blazegraphViewsEventManifest
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext.keywords
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Event
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveConfiguredCodec
+import io.circe.parser.decode
+import io.circe.syntax._
+import io.circe.{Codec, Decoder}
+
+import java.nio.charset.StandardCharsets
+import scala.annotation.nowarn
+
+/**
+  * A json serializer for Blazegraph plugin [[Event]] types.
+  */
+@nowarn("cat=unused")
+@SuppressWarnings(Array("UnusedMethodParameter"))
+class EventSerializer(system: ExtendedActorSystem) extends SerializerWithStringManifest {
+  override def identifier: Int = 453224
+
+  override def manifest(o: AnyRef): String = o match {
+    case _: BlazegraphViewEvent => blazegraphViewsEventManifest
+    case _                      =>
+      throw new IllegalArgumentException(
+        s"Unknown event type '${o.getClass.getCanonicalName}', expected BlazegraphViewEvent"
+      )
+  }
+
+  override def toBinary(o: AnyRef): Array[Byte] = o match {
+    case e: BlazegraphViewEvent => e.asJson.noSpaces.getBytes(StandardCharsets.UTF_8)
```

Comment:
we should probably use a printer:
```scala
   private val printer: Printer = Printer. noSpaces.copy(dropNullValues = true)
   printer.print(e.asJson)
```
To avoid printing null values (we should do that in all `EventSerializer`, in this case we will gain few bytes here and there)